packages/rule-stylesheet-limits/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "outDir": "dist"
```

Comment:
Can you add the option `"strict": true`? if there is errors just in your rule, fix them and keep the option, if there is a lot of errors in `sonarwhal` that are not easy to fix, then remove it.

The idea is adding the strict option step by step until the whole project use it.

Thanks!!!

Reply:
Yes, I'll add this in my next update.